docs/public/using/backend/rpcapi-reference.md

```diff
@@ -1436,6 +1439,206 @@ If the `batch_id` is undefined the following error is returned:
 }
 ```
 
+
+### API method: `batch_status`
+
+This method replaces deprecated method
+[API method: get_batch_job_result](#api-method-get_batch_job_result).
+
+Returns the number of created, running and finished *tests*. Optionally it also
+returns [*test id*][Test id] objects (created, running or finished) of a
+*batch test*.
+
+Example valid request:
```

Comment:
I have two suggestions that I believe could improve this list of examples.
1. Condense it to one minimal example and one more comprehensive example. This method is quite simple.
2. Show example responses right after their corresponding example requests. That way we get the explanation for the circumstance of the response for free.

I expect it would take slightly more effort to construct the examples this way, but we'd bring the information density up, and I believe the reading experience would be more rewarding.

Reply:
I think the examples are important, but they could be moved to an appendix at the end.

Reply:
Having matching query and response together is a good suggestion.

Reply:
Fixed.